src/linter/utils/semver.mjs

```diff
@@ -0,0 +1,13 @@
+/**
+ * Validates a semver version string
+ *
+ * @param {string} version
+ * @returns {boolean}
+ */
+export const validateVersion = version => {
+  // https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+  const regex =
```

Comment:
We could use `semver` package, instead of maintaining our own regex. I do recall we already use semver package here? And it has a method to validate if a string is a semver.